pydicom/data/base.py

```diff
@@ -0,0 +1,51 @@
+"""pydicom data manager"""
+#
+# Copyright (c) 2008-2012 Darcy Mason
+# Copyright (c) 2017 pydicom AUTHORS
+# This file is part of pydicom, released under a modified MIT license.
+#    See the file LICENSE included with this distribution, also
+#    available at https://github.com/pydicom/pydicom
+#
+
+import fnmatch
+import os
+
+from .utils import get_files
+
+
+def get_datadir():
+    return os.path.abspath(os.path.dirname(__file__))
+
+
+def get_testdata_base():
+    return os.path.join(get_datadir(), 'test_files')
+
+
+def get_charset_base():
+    return os.path.join(get_datadir(), 'charset_files')
+
+
+'''
+
+Data get functions.
+
+For each of the below, a complete list of files is returned.
+Note the distinction between "get" (returning a list) and
+what might be interpreted as reading in the files (load).
+These functions serve to only provide the paths.
+
+Optionally, the user can specify a pattern to filter by.
+
+'''
+
+
+def get_charset_files(pattern=None):
```

Comment:
```py
from pydicom.data import get_testdata_base
import glob
glob.glob(get_testdata_base()+'/**/*', recursive=True)
```
I'm not saying that this should be the end of the end of what we are trying to do with `pydicom/data/base.py`. Far from this. What I'm saying is that at this stage I don't have a clear idea of the direction that `get_files` and its helper functions should take. Therefore I would implement something like:

```py
def test_needing_an_awesome_dcm():
        filename = glob.glob(get_testdata_base()+'/**/my_awesome.dcm', recursive=True)
        ... bla bla
```